Dockerfile

```diff
@@ -60,7 +60,7 @@ RUN pip3 install --user bsddb3==6.2.7
 WORKDIR /sio2/oioioi
 
 COPY --chown=oioioi:oioioi setup.py requirements.txt ./
-RUN pip3 install -r requirements.txt --user
+RUN pip3 install -r requirements.txt --user filetracker[server]
```

Comment:
I'm not sure if this is the best approach:
- the version range for `filetracker` is not specified.
- In `oioioi`, there is a `setup.py` file where this requirement is defined. 

I have already addressed this issue in the #434 , due to the Python update.